libxenon/drivers/xenon_nand/xenon_sfcx.c

```diff
@@ -586,12 +604,24 @@ int try_rawflash(char *filename)
 
 	printf(" * Checking NAND File to be of matching type...\n");
 
-	if (rawflash_checkImage(f) != 0) {
-		printf(" ! Bad Image for this console... Please replace the file and try again...\n");
-		return -1;
+	if (rawflash_checkImage(f) != 0)
+   {
```

Comment:
```suggestion
    {
```
Please correct the indentation so it lines up with the if